src/Tests/dotnet.Tests/CommandTests/ToolInstallGlobalOrToolPathCommandTests.cs

```diff
@@ -307,7 +330,7 @@ public void WhenRunWithInvalidVersionItShouldThrow()
         [Fact]
         public void WhenRunWithExactVersionItShouldSucceed()
         {
-            ParseResult result = Parser.Instance.Parse($"dotnet tool install -g {PackageId} --version {PackageVersion}");
+            ParseResult result = Parser.Instance.Parse($"dotnet tool install -g {PackageId} --version {PackageVersion} --verbosity minimal");
```

Comment:
Having reverted the default, do you still need to explicitly set verbosity to minimal in all these tests?

Reply:
Yes. Because by default verbosity is quiet (first element in the verbosity Enum), which has no output unless there is an error.